src/app.js

```diff
@@ -180,6 +180,10 @@ export function app(props) {
   }
 
   function patch(parent, element, oldNode, node, isSVG, nextSibling) {
+    if (oldNode === node) {
+      console.log("memoized...")
```

Comment:
I love `console.log` as much as the next guy, but perhaps it's not needed here?

Reply:
was just to help me verify that it was actually running that memoize case, trying to rack down the uncovered line #266 in coverage report LOL